larpandora/LArPandoraInterface/LArPandoraHelper.h

```diff
@@ -9,6 +9,10 @@
 
 #include "art/Framework/Principal/Event.h"
 
+#include "larcore/Geometry/Geometry.h"
+#include "larcorealg/Geometry/GeometryCore.h"
+#include "larcorealg/Geometry/PlaneGeo.h"
```

Comment:
It looks like the interfaces of these header files are not referenced in this header file, in which case I would move these includes into the source file in which they are needed (avoids the public interface carrying around extra header files which can slow builds, among other things).

Reply:
Moved!